src/Slices/ResourceDetails/UI/Tabs/AttributesTab/AttributesTab.tsx

```diff
@@ -1,32 +1,32 @@
 import React from "react";
 import { Card, CardBody } from "@patternfly/react-core";
-import { Query, RemoteData } from "@/Core";
+import { Details } from "@/Core/Domain/Resource/Resource";
```

Comment:
The import path makes me guess there's no index file

Reply:
If there's an index file, you should be able to do 
```suggestion
import { Details } from "@/Core/Domain/Resource";
```